src/main/java/pl/jkap/sozzt/contract/ContractController.java

```diff
@@ -0,0 +1,45 @@
+package pl.jkap.sozzt.contract;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import org.springframework.web.bind.annotation.*;
+
+import pl.jkap.sozzt.contract.domain.ContractFacade;
+import pl.jkap.sozzt.contract.dto.ContractDto;
+
+import javax.validation.constraints.Min;
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+public class ContractController {
+
+    private final ContractFacade contractFacade;
+
+    @GetMapping("/contracts/{id}")
+    @ResponseStatus(HttpStatus.OK)
+    public ContractDto getContract(@PathVariable long id) {
+        return contractFacade.getContract(id);
+    }
+
+    @PostMapping("/contracts")
+    @ResponseStatus(HttpStatus.CREATED)
+    public ContractDto addContract(@RequestBody ContractDto contractDto) {
+        return contractFacade.addContract(contractDto);
+    }
+
+    @PutMapping("/contracts/confirm_step/{id}")
```

Comment:
`"/contracts/confirm_step/{id}"` I think that the id is the contract id (maybe change variable so it is known at first glance), if so the endpoint should be `"/contracts/{id}/confirm-step"`. You should not use the `_` sign in multiwords, the rule is `-`